lib/Auth/Process/COmanageDbClient.php

```diff
@@ -0,0 +1,480 @@
+<?php
+
+/**
+ * COmanage authproc filter.
+ *
+ * This class is the authproc filter to get attributes from rest api
```

Comment:
Update description

Reply:
Changes made in "Update COmanageDbClient description and authors" commit